service/src/main/resources/migrations.txt

```diff
@@ -260,4 +260,3 @@ V259__early_childhood_education_secretary_acl.sql
 V260__drop_irregular_daily_service_times_constraint.sql
 V261__invoice_correction_person_reference_on_delete_cascade.sql
 V262__child_consent_on_delete_cascade.sql
-V263__child_absences_function.sql
```

Comment:
You must keep the migration file and this line, otherwise future migrations will fail hard.